svcomp/src/main/java/com/dat3m/svcomp/SVCOMPRunner.java

```diff
@@ -76,30 +78,34 @@ public static void main(String[] args) throws Exception {
             return;
         }
 
-		if(Arrays.stream(args).noneMatch(a -> supportedFormats.stream().anyMatch(a::endsWith))) {
-			throw new IllegalArgumentException("Input program not given or format not recognized");
-		}
-		if(Arrays.stream(args).noneMatch(a -> a.endsWith(".cat"))) {
-			throw new IllegalArgumentException("CAT model not given or format not recognized");
-		}
-		File fileModel = new File(Arrays.stream(args).filter(a -> a.endsWith(".cat")).findFirst().get());
-		String programPath = Arrays.stream(args).filter(a -> supportedFormats.stream().anyMatch(a::endsWith)).findFirst().get();
-		File fileProgram = new File(programPath);
-
-		String[] argKeyword = Arrays.stream(args)
-		.filter(s->s.startsWith("-"))
-		.toArray(String[]::new);
-		Configuration config = Configuration.fromCmdLineArguments(argKeyword);
-		SVCOMPRunner r = new SVCOMPRunner();
-		config.recursiveInject(r);
+        if(Arrays.stream(args).noneMatch(a -> supportedFormats.stream().anyMatch(a::endsWith))) {
+            throw new IllegalArgumentException("Input program not given or format not recognized");
+        }
+        if(Arrays.stream(args).noneMatch(a -> a.endsWith(".cat"))) {
+            throw new IllegalArgumentException("CAT model not given or format not recognized");
+        }
+        File fileModel = new File(Arrays.stream(args).filter(a -> a.endsWith(".cat")).findFirst().get());
+        String programPath = Arrays.stream(args).filter(a -> supportedFormats.stream().anyMatch(a::endsWith)).findFirst().get();
+        File fileProgram = new File(programPath);
+
+        String[] argKeyword = Arrays.stream(args)
+            .filter(s->s.startsWith("-"))
+            .toArray(String[]::new);
+        Configuration config = Configuration.fromCmdLineArguments(argKeyword);
+        SVCOMPRunner r = new SVCOMPRunner();
+        config.recursiveInject(r);
+
+        if(r.property == null) {
+            System.out.println("UNKNOWN");
+            return;
+        }
```

Comment:
Can we also log a warning that the input property is not supported so we can better distinguish between UNKNOWN due to unrolling and UNKNOWN due to unsupported property?

Reply:
Done